packages/cra-template/template.json

```
@@ -15,6 +15,7 @@
       "@nimblehq/eslint-config-nimble-react": "^1.0.0",
       "@nimblehq/stylelint-config-nimble": "^1.0.0",
       "@testing-library/cypress": "8.0.2",
+      "@testing-library/dom": "^8.20.0",
```

if we are fixing version, maybe set it without `^`? as we are doing in some other places
```suggestion
      "@testing-library/dom": "8.20.0",
```

---

Done in e5e43752140b3aa69bc715183a05dc5b511eee5c 👍